### PR TITLE
Add margin to Tasker plugin UI

### PR DESCRIPTION
**Pull request by jacobherrington**
# Description

As the first time I've ever run an Android application locally or touched a Kotlin file, I'll call this a win.

I plan to follow up on additional improvements to this UI to make it resemble the filters tab, however, that will likely take me a bit as a novice mobile developer so I wanted to push this minor tweak first :)

Related to #471 (does not close)

# Checklist

- [x] Should this change be included in the release notes? If yes, please add a line in CHANGELOG.md (no)
- [x] Have you tested in landscape?
- [ ] Have you tested accessibility with TalkBack?
- [ ] Have you tested in different themes?
- [ ] Does the change work with a large display font?
- [ ] Are all the strings localized?
- [ ] Could you have written any new tests?
- [ ] Did you include Compose previews with any components?

### CODE REVIEW

**Comment by jacobherrington** [comment]
@mchowning this is my first ever change to a native Android app, I'd love to follow up on your other suggestions, I might ping you for help in the issue as I'm working on it. :)

File: modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/base/ComposableBase.kt [hunk 1]
```diff
@@ -164,7 +164,9 @@ fun ComposableTaskerInputFieldList(
     fieldContents: List<TaskerInputFieldState.Content<*>>,
     onFinish: () -> Unit
 ) {
-    Box(modifier = Modifier.fillMaxHeight()) {
+    Box(modifier = Modifier
+            .padding(10.dp)
```
Comment (TylerCarberry) [inline modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/base/ComposableBase.kt:168]: nitpick, but material guidelines suggest using an 8dp grid https://m1.material.io/layout/metrics-keylines.html#metrics-keylines-baseline-grids

Comment (jacobherrington) [reply modules/features/taskerplugin/src/main/java/au/com/shiftyjelly/pocketcasts/taskerplugin/base/ComposableBase.kt]: TIL thanks for sharing.

**Review by mchowning — APPROVED** [review]
This screen is definitely moving in the right direction. Thanks @jacobherrington !

> As the first time I've ever run an Android application locally or touched a Kotlin file, I'll call this a win.

I agree! 😄 

> I plan to follow up on additional improvements to this UI to make it resemble the filters tab, however, that will likely take me a bit as a novice mobile developer so I wanted to push this minor tweak first :)

That sounds like a good plan. 👍

**Pull request merged**